tests/base/managers/entity.py

```diff
@@ -591,3 +591,37 @@ def test_mgr_entity_search_translation(entity_test_search):
     assert search('first translation') == [entities[0]]
     assert search('second translation') == [entities[1]]
     assert search('third translation') == [entities[2]]
+
+
+@pytest.mark.django_db
+def test_lookup_collation(resource0, locale0, translation_factory):
+    """
+    Filter translations according to collation.
+    """
+    entity = Entity.objects.create(
+        resource=resource0,
+        string="string")
+    batch_kwargs = [dict(entity=entity, locale=locale0, string=s) 
+        for s in [u'this is string',
+            u'this is STRİNG',
+            u'this is Strıng',
+            u'this is StrInG',
+            u'this is sTriNg']]
+    translations = translation_factory(
+        batch_kwargs=batch_kwargs)
+    assert (
```

Comment:
one last request 8)

would you mind adding a comment before each of these asserts explaining what its testing